unitTests/ui/DynamicFacet/DynamicFacetRangeTest.ts

```diff
@@ -80,13 +80,20 @@ export function DynamicFacetRangeTest() {
       expect(test.cmp.facetType).toBe(FacetType.numericalRange);
     });
 
-    it(`when not setting a valueFormat option
-      should set it number`, () => {
+    it(`when not setting a valueFormat option with a field containing "date"
+      should set it to "date"`, () => {
+      options.field = '@cooldatefield';
+      initializeComponent();
+      expect(test.cmp.options.valueFormat).toBe(DynamicFacetRangeValueFormat.date);
+    });
+
+    it(`when not setting a valueFormat option with a field not containing "date"
+      should set it to "number"`, () => {
       expect(test.cmp.options.valueFormat).toBe(DynamicFacetRangeValueFormat.number);
     });
 
     it(`when not setting an invalid valueFormat option
-      should set it number`, () => {
+      should set it to "number"`, () => {
       options.valueFormat = 'hello' as DynamicFacetRangeValueFormat;
       initializeComponent();
       expect(test.cmp.options.valueFormat).toBe(DynamicFacetRangeValueFormat.number);
```

Comment:
Is there a test checking that when the value is not specified, it defaults to `date` to capture the first half of the `if` on line 63 in DynamicFacetRange?

Reply:
Yes line 88